crates/sui-types/src/transaction.rs

```diff
@@ -365,7 +374,12 @@ impl EndOfEpochTransactionKind {
             }
             Self::RandomnessStateCreate => vec![],
             Self::DenyListStateCreate => vec![],
-            Self::BridgeStateCreate => vec![],
+            Self::BridgeStateCreate(_) => vec![],
+            Self::BridgeCommitteeInit(bridge_version) => vec![InputObjectKind::SharedMoveObject {
+                id: SUI_BRIDGE_OBJECT_ID,
+                initial_shared_version: *bridge_version,
+                mutable: true,
+            }],
```

Comment:
why don't we need 0x5 here?

Reply:
unresolve this because it doesn't break anything is concerning. @patrickkuo  do you have some ideas?

Reply:
I had a look but couldn't figure out if we are using the `input_objects` function during execution, I guess this is only used for read operations (display, rpc etc), hence not breaking anything. 

Perhaps @tzakian or @tnowacki might know more?